src/main/java/com/patreon/PatreonAPI.java

```diff
@@ -54,6 +54,7 @@ public PatreonAPI(String accessToken) {
 
     ObjectMapper objectMapper = new ObjectMapper();
     objectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE);
+    objectMapper.configure(com.fasterxml.jackson.databind.DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Are we intentionally allowing unknown properties here? Just wondering what the use-case for them is. I'm guessing it's because we don't want this to break if the API changes, but want to make sure I understand :smiley_cat:

Reply:
Yup. Older versions of the client libs shouldn't blow up if we add fields.